src/network/networkd-bridge-mdb.c

```diff
@@ -355,10 +385,16 @@ int config_parse_mdb_group_address(
         if (r < 0)
                 return log_oom();
 
-        r = in_addr_from_string_auto(rvalue, &mdb->family, &mdb->group_addr);
-        if (r < 0) {
-                log_syntax(unit, LOG_WARNING, filename, line, r, "Cannot parse multicast group address: %m");
-                return 0;
+        r = parse_ether_addr(rvalue, &mdb->l2_addr);
+        if (r >= 0)
+                mdb->type = BRIDGE_MDB_ENTRY_TYPE_L2;
+        else {
+                r = in_addr_from_string_auto(rvalue, &mdb->family, &mdb->group_addr);
+                if (r < 0) {
+                        log_syntax(unit, LOG_WARNING, filename, line, r, "Cannot parse multicast group address as either L2 MAC, IPv4 or IPv6: %m");
```

Comment:
if you touch this, please suffix he error message with ", ignoring". i.e. "… as ither L2 MAC, IPv4 or IPv6, ignoring: %m".

i.e. if we warn about something but then proceed without it we always want to make clear that this is something we ignored.

Reply:
Agree, fixed.